contracts/SpokePool.sol

```diff
@@ -647,6 +647,48 @@ abstract contract SpokePool is
      *         USS DEPOSITOR FUNCTIONS          *
      ********************************************/
 
+    /**
+     * @notice Request to bridge input token cross chain to a destination chain and receive a specified amount
+     * of output tokens. The fee paid to relayers and the system should be captured in the spread between output
+     * amount and input amount when adjusted to be denominated in the input token. A relayer on the destination
+     * chain will send outputAmount of outputTokens to the recipient and receive inputTokens on a repayment
+     * chain of their choice plus system fees. Therefore, the fee should account for destination fee transaction costs,
+     * the relayer's opportunity cost of capital while they wait to be refunded following an optimistic challenge
+     * window in the HubPool, and the system fee that they'll be charged.
+     * @dev On the destination chain, the hash of the deposit data will be used to uniquely identify this deposit, so
+     * modifying any params in it will result in a different hash and a different deposit. The hash will comprise
+     * all parameters to this function along with this chain's chainId(). Relayers are only refunded for filling
+     * deposits with deposit hashes that map exactly to the one emitted by this contract.
+     * @param depositor The account credited with the deposit who can request to "speed up" this deposit by modifying
+     * the output amount, recipient, and message.
+     * @param recipient The account receiving funds on the destination chain. Can be an EOA or a contract. If
+     * the output token is the wrapped native token for the chain, then the recipient will receive native token if
+     * an EOA or wrapped native token if a contract.
+     * @param inputToken The token pulled from the caller's account and locked into this contract to
+     * initiate the deposit. The equivalent of this token on the relayer's repayment chain of choice will be sent
+     * as a refund. If this is equal to the wrapped native token then the caller can optionally pass in native token as
+     * msg.value, as long as msg.value = inputTokenAmount.
+     * @param outputToken The token that the relayer will send to the recipient on the destination chain. Must be an
+     * ERC20.
+     * @param inputAmount The amount of input tokens to pull from the caller's account and lock into this contract.
+     * This amount will be sent to the relayer on their repayment chain of choice as a refund following an optimistic
+     * challenge window in the HubPool, plus a system fee.
```

Comment:
Same as above - as I read it, it seems to imply that the relayer receives `inputAmount` of `inputToken` + some additional fee, when instead they should receive `inputAmount - systemFee` of `inputToken`.